src/server.h

```diff
@@ -1743,6 +1744,7 @@ struct redisServer {
     int lazyfree_lazy_server_del;
     int lazyfree_lazy_user_del;
     int lazyfree_lazy_user_flush;
+    int busy_job;
```

Comment:
since this is now specific to modules, let's call it `busy_module` (should be set as soon at the timeout is reached, and remain set until the context is destroyed)